integration_tests/integration/court-register-list.spec.ts

```diff
@@ -187,4 +198,74 @@ context('Court register - court list navigation', () => {
         expect(page1Requests[0].request.url).to.contain('courtTypeIds=COU')
       })
   })
+  it('Will remove the active filter when cancelling via the tag', () => {
+    // IndexPage.verifyOnPage().courtRegisterLink().click()  -  TODO Will need this when plumbing in the paged court list and removign the all courts list
+    IndexPage.verifyOnPage()
+    cy.visit('/court-register/paged') // TODO and this will need removing
+    const page = AllCourtsPagedPage.verifyOnPage()
+
+    // Filter on open courts
+    page.showFilterButton().click()
+    page.openFilter().click()
+    page.applyFilterButton().click()
+    page.showFilterButton().click()
+    page.openFilter().should('be.checked')
+    // Cancel the open courts by clicking on the filter tag
+    page.cancelOpenFilter().click()
+    page.showFilterButton().click()
+    // Check that the filter is no longer applied
+    page.cancelOpenFilter().should('not.exist')
+    page.openFilter().should('not.be.checked')
+    page.allFilter().should('be.checked')
+    cy.url().should('not.contain', 'active=true')
```

Comment:
I've tried to make these tests more readable by tidying up the function names and adding some comments to convey the intent of the test.  I think I will refactor them in a future PR to use some kind of test DSL so the comments aren't required.